requirements.txt

```diff
@@ -9,4 +9,4 @@ selenium==4.5.0
 filelock==3.8.0
 packaging==21.3
 prettytable==3.5.0
-bzt==1.16.3     # make sure Dockerfile bzt version is the same
+bzt==1.16.18     # make sure Dockerfile bzt version is the same
```

Comment:
This comment should be removed if we move from the base Taurus image to python slim.